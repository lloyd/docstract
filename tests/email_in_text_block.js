/** Emails should not be interpreted as tags, so lloyd@hilaiel.com
 * shouldn't be broken at @@hilaiel */
