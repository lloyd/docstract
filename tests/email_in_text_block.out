{
  "desc": "Emails should not be interpreted as tags, so lloyd@hilaiel.com\nshouldn't be broken at @hilaiel", 
  "filename": "tests/email_in_text_block.js", 
  "module": "email_in_text_block"
}
